docker/install/install_parsl.sh

```diff
@@ -1,7 +1,54 @@
 #!/bin/env bash
 
+# These can be customized to suit individual needs
+DEFAULT_GCC_VERSION=$(/usr/bin/gcc --version | head -1 | sed -e 's/([^()]*)//g' | awk '{print $2}')  # Version of system default gcc
+DEFAULT_COMPILER="gcc@${DEFAULT_GCC_VERSION}"  # Default system compiler used to build newer gcc
+
+SPACK_ENV_NAME="flux"            # Name of spack environment to create
+SPACK_ENV_COMPILER="gcc@9.4.0"  # Compiler to use to build the spack environment
+TARGET_ARCH_OPT="target=x86_64"  # Compiler architecture build target
+
+################################################################################
+# help                                                                         #
+################################################################################
+help()
+{
+   # Display help
+   echo "Installs parsl into flux Spack environment"
+   echo
+   echo "Usage: install_parsl.sh"
+   echo
+}
+
+# Get the location of Spack so we can update permissions
+if [[ $(which spack 2>/dev/null) ]]; then
+  SPACK_DIR=$(dirname $(dirname $(which spack)))
+else
+  echo "Cannot find Spack."
+  echo
+  echo "Please install Spack and/or source Spack's environment setup script: .../spack/share/setup-env.sh"
+  echo
+  exit 1
+fi
+
+set -eu
+
+# Configure spack
+spack config add concretizer:unify:true
+spack config add concretizer:reuse:true
+spack config add config:db_lock_timeout:300
+spack config add config:install_tree:padded_length:128
+
+# Configure spack environment
+spack env create ${SPACK_ENV_NAME} || true
+spack env activate ${SPACK_ENV_NAME}
+
+# Re-install pip to update the view and make it reappear in the flux environment
```

Comment:
Yes, this is the same thing.  For some strange reason, pip disappears from the view for the Spack flux environment.  So, I have to reinstall it to get it back into the view.  Again, it doesn't actually rebuild it; it just finds the build it already did and re-adds it to the view so that the Flux environment "sees" it.